manifests/deis-builder-rc.yaml

```diff
@@ -40,10 +58,6 @@ spec:
             - name: builder-key-auth
               mountPath: /var/run/secrets/api/auth
               readOnly: true
-            # not currently running minio with SSL support. see https://github.com/deis/minio/pull/22 for more detail
```

Comment:
oh I meant like removing the commented-out code entirely rather than leaving it as a comment